studio/app/common/dataclass/polar.py

```diff
@@ -33,14 +32,23 @@ def output_path(self) -> OutputPath:
 
     def save_plot(self, output_dir):
         for i in range(len(self.data)):
-            fig = px.line_polar(
-                r=self.data[i],
-                theta=self.columns,
-                direction="counterclockwise",
-                start_angle=0,
-                line_close=True,
+            theta = linspace(0, 2 * pi, len(self.columns))  # Convert theta to radians
+            plt.figure()
+            ax = plt.subplot(111, polar=True)
+            ax.plot(theta, self.data[i], color="#636EFA")
+            ax.plot(
+                [theta[-1], theta[0]],
+                [self.data[i][-1], self.data[i][0]],
+                color="#FFA15A",
+                linestyle="-",
+                linewidth=2,
             )
-            plot_file = join_filepath([output_dir, f"{self.file_name}_{i}.png"])
-            pio.write_image(fig, plot_file)
-
-            save_thumbnail(plot_file)
+            ax.set_theta_direction(1)  # Clockwise
+            ax.tick_params(axis="x", rotation="auto")
+            ax.set_facecolor("white")
+            fig = plt.gcf()
+            fig.set_facecolor("white")
+            plt.tight_layout()
+            plot_file = join_filepath([output_dir, f"{self.filename}{i}.png"])
+            plt.savefig(plot_file, bbox_inches="tight")
```

Comment:
There seems to be a slight difference in the results of POLAR PLOT from the version before the modification.
*Refer to the figure below.
*There seems to be a slight difference not only in the appearance but also in the information (angle) drawn.
*In other plot images, there seems to be no change in the information drawn before and after the modification.
*Which data content is correct, v1 or v2?

- v1
  - ![tuning_curve_polar0](https://github.com/user-attachments/assets/078b3398-c3b6-4a07-9dfd-3f49b86ec33c)

- v2
  - ![tuning_curve_polar0](https://github.com/user-attachments/assets/775abe3a-18ba-4457-9169-0399e6b4ceae)

Reply:
I think the new format it actually correct and the previous approach was slightly mistaken. 

The new polar plot (second style) is appropriate if:

- The data represents cyclical/periodic phenomena
- The data is directional data (like wind direction, angles)
- The relationship between the first and last data points is meaningful
- You want to emphasize patterns that repeat or come full circle

The old line plot (first style) is more appropriate if:

- The data represents a linear sequence
- We want to show clear trends over time or another continuous variable
- Direct comparison of values is important
- You need to emphasize specific peaks or troughs in the data

Because this is a polar plot to show preference for a specific angle, and because there is a relationship between 0 and 360 degrees, then I believe the new approach is more correct. 

新しいフォーマットは実際には正しく、以前のアプローチは若干誤っていたと思います。

新しい極座標プロット（2番目のスタイル）は、以下の場合に適切です。

- データが周期的/周期的現象を表している
- データが方向性データ（風向、角度など）である
- 最初のデータポイントと最後のデータポイントの間の関係が意味を持つ
- 繰り返されるパターンや循環するパターンを強調したい

データが直線的なシーケンスを表している場合、古い線プロット（第1のスタイル）の方が適切です。

- データが直線的な順序を表している
- 時間経過や別の連続変数における明確な傾向を示したい
- 値の直接比較が重要である
- データにおける特定のピークまたは谷を強調する必要がある

これは特定の角度に対する好みを表す極座標プロットであり、0度と360度には関係があるため、新しいアプローチの方がより適切であると思います。

Reply:
After checking some neuroscience papers I think i have understood the issue. 
Previous approach plotted in degrees. 
Matlab expects radians data for polar plots. 
Radians is more correct in most cases, however, in neuroscience it is also common ([example](https://www.jneurosci.org/content/32/26/9073)) to plot in degrees (not radians). Both are correct, however, we should match the expectations of the researchers. 

Solution: convert to radians for plotting with matplotlib, but use custom labels to display data in degrees for easy interpretation

Reply:
Plotly:
![test_plot_0_plotly](https://github.com/user-attachments/assets/f4280f59-fbbd-4a50-844c-ac4daa2b08d6)
Matplotlib:
![test_plot_0_matplotlib](https://github.com/user-attachments/assets/221eecd4-6880-4d09-80ac-c66493db1aa4)

Reply:
With the example data:
{"data": [-0.008712863097971257, -0.0015235116947858174, -0.004882731632390502, 0.011767370797716614, 0.001840553248896093, -0.008743215513716152, 0.00328831169626409, 0.004903902409403305], "columns": [0.0, 45.0, 90.0, 135.0, 180.0, 225.0, 270.0, 315.0], "file_name": "tuning_curve_polar", "index": 3}

![test_plot_0_plotly](https://github.com/user-attachments/assets/08c58e16-454e-4446-8601-84700ba4a6fe)
![test_plot_0_matplotlib](https://github.com/user-attachments/assets/c2c1a266-3ad4-4ae0-acd6-ea07ff09c7e0)

Reply:
- Good
  - The revised version confirms that the data displayed seems to be in close agreement.

- Problems
  - [ ] The polar image has become larger again, so please adjust it to the original image size.
  - [ ] The thumbnail image is grainy and difficult to see. Improvement is required.
    - ![tuning_curve_polar_0_matplotlib thumb](https://github.com/user-attachments/assets/3d5b5917-14be-4e1e-87ff-d74bbd8a2cf9)
    - ![tuning_curve_polar_1_matplotlib thumb](https://github.com/user-attachments/assets/52c3289e-31e6-4853-b177-0631fa023a5f)

    -